src/amplitude-client.js

```diff
@@ -1269,6 +1269,13 @@ AmplitudeClient.prototype._logEvent = function _logEvent(
     eventProperties = eventProperties || {};
     groups = groups || {};
     groupProperties = groupProperties || {};
+    const plan = _isObservePlanSet(this)
+      ? {
+          branch: this.options.plan.branch || null,
```

Comment:
Why `null` here versus `undefined`? `null` will pass a (unnecessary?) value versus `undefined` which will omit it from the payload.